packages/core/useWheel/index.ts

```diff
@@ -0,0 +1,99 @@
+import { reactive, ref } from 'vue-demi'
+import type { MaybeRef } from '@vueuse/shared'
+import { noop, useDebounceFn, useThrottleFn } from '@vueuse/shared'
+import { useEventListener } from '../useEventListener'
+
+export interface UseWheelOptions {
+  /**
+   * Throttle time for scroll event, it’s disabled by default.
+   *
+   * @default 0
+   */
+  throttle?: number
+
+  /**
+   * The check time when scrolling ends.
+   * This configuration will be setting to (throttle + idle) when the `throttle` is configured.
+   *
+   * @default 200
+   */
+  idle?: number
+
+  /**
+   * Trigger it when scrolling.
+   *
+   */
+  onScroll?: (e: Event) => void
+
+  /**
+   * Trigger it when scrolling ends.
+   *
+   */
+  onStop?: (e: Event) => void
+
+  /**
+   * Listener options for scroll event.
+   *
+   * @default {capture: false, passive: true}
+   */
+  eventListenerOptions?: boolean | AddEventListenerOptions
+}
+
+export function useWheel(
+  element: MaybeRef<HTMLElement | SVGElement | Window | Document | null | undefined>,
+  options: UseWheelOptions = {},
+) {
+  const {
+    throttle = 0,
+    idle = 200,
+    onStop = noop,
+    eventListenerOptions = {
+      capture: false,
+      passive: true,
+    },
+  } = options
+
+  const isWheeling = ref(false)
```

Comment:
I think we should either rename this to `isScrolling` or change `onScroll` to `onWheel`. Although I think "onWheeling" sounds a little funny haha